save-demo/src/main/kotlin/com/saveourtool/save/demo/controller/ManagementController.kt

```diff
@@ -0,0 +1,53 @@
+package com.saveourtool.save.demo.controller
+
+import com.saveourtool.save.demo.NewDemoToolRequest
+import com.saveourtool.save.demo.entity.GithubRepo
+import com.saveourtool.save.demo.entity.Snapshot
+import com.saveourtool.save.demo.entity.Tool
+import com.saveourtool.save.demo.service.GithubDownloadToolService
+import com.saveourtool.save.demo.service.GithubRepoService
+import com.saveourtool.save.demo.service.SnapshotService
+import com.saveourtool.save.demo.service.ToolService
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import reactor.core.publisher.Mono
+import reactor.kotlin.core.publisher.toMono
+import reactor.kotlin.core.util.function.component1
+import reactor.kotlin.core.util.function.component2
+
+/**
+ * Internal controller that allows to add tools to db
+ */
+@RestController
+@RequestMapping("/demo/internal")
```

Comment:
do we need prefix `/demo` in demo project?

Reply:
I think it might be a good sign that the request needs to be sent to `save-demo`

Reply:
a good sign in gateway, but not in application (we don't have prefix for preprocessor and orchestrator).
but up to you